### ISSUE TITLE
Port to de0-nano

### ISSUE THREAD

**Issue by howard0su** [PulseRain/Reindeer#20]
Can we port over a board like de0-nano which has SDRAM onboard? We may need create a internal bus and I-Cache and D-Cache.

**Comment by PulseRainmaker** [comment]
Yes, we can. In fact, the Reindeer has been ported to a similar board (Step CYC10), which uses SDRAM. 
See https://github.com/PulseRain/Reindeer_Step

Right now, there is no cache supported for the SDRAM yet.
